modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -168,6 +169,55 @@ final case class BuildOptions(
       }
     }
 
+  def latestSupportedScalaVersion(prefixScalaVersion: Option[String]): coursier.core.Version = {
+
+    // used when downloading is fail
+    lazy val defaultStableScalaVersions = StableScalaVersion(
+      version,
+      Seq(defaultScala212Version, defaultScala213Version, defaultScalaVersion)
+    )
+
+    val supportedScalaVersionsUrl = scalaOptions.scalaVersionsUrl
+
+    val task = {
+      val art = Artifact(supportedScalaVersionsUrl).withChanging(true)
+      finalCache.file(art).run.flatMap {
+        case Left(e) => Task.fail(e)
+        case Right(f) => Task.delay {
+            val content = os.read.bytes(os.Path(f, Os.pwd))
+            content
+          }
+      }
+    }
+
+    val launchersTask = finalCache.logger.using(Task.gather.gather(Seq(task)))
```

Comment:
No need to use `gather` if we only have one task:
```suggestion
    val launchersTask = finalCache.logger.using(task)
```